src/main/fc/fc_init.c

```diff
@@ -546,7 +546,7 @@ void init(void)
 #endif
 #ifdef USE_MSP_OSD
         if (!osdDisplayPort) {
-            osdDisplayPort = mspOsdDisplayPortInit();
+            osdDisplayPort = mspOsdDisplayPortInit(osdConfig()->video_system);
```

Comment:
Shouldn't need to pass this as a parameter `mspOsdDisplayPortInit()` should be able to access `osdConfig()->video_system` directly.